Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -435,6 +436,8 @@ extension TelemetryWrapper {
             GleanMetrics.Tabs.open[value].add()
         case (.action, .close, .tab, let value, _):
             GleanMetrics.Tabs.close[value].add()
+        case (.action, .tap, .addNewTabButton, _, _):
+            GleanMetrics.Tabs.addNewTabPressed.add()
```

Comment:
Based on the name in the metrics.yaml, this should have been:
```suggestion
            GleanMetrics.Tabs.addNewTabButton.add()
```